entc/gen/template/client.tmpl

```diff
@@ -251,6 +254,7 @@ func (c *{{ $client }}) Hooks() []Hook {
 {{ end }}
 {{ end }}
 
-{{/* A template that can be overrided in order to add additional fields to the client.*/}}
+{{/* A template that can be overridden in order to add additional fields to the client.*/}}
 {{ define "client/fields/additional" }}{{ end }}
 
+{{ define "client/query/init" }}{{ end }}
```

Comment:
Add a template comment?

Reply:
Will add comment

I actually dont like the name init because i find it a bit mis leading. Because the actual template is adding fields to struct init. What about `define "client/query/init/fields"`

Reply:
yes